docs/ref/django-admin.txt

```diff
@@ -979,6 +979,20 @@ Lets you pass a command as a string to execute it as Django, like so::
 
     django-admin shell --command="import django; print(django.__version__)"
 
+.. versionadded:: 1.11
```

Comment:
I don't think this is necessarily a 'versionadded', more of a 'versionchanged' since you could already pass code in on stdin